clang/lib/CodeGen/CGLoopInfo.h

```diff
@@ -363,13 +354,8 @@ class LoopInfoStack {
   /// Set value of an initiation interval for the next loop pushed.
   void setSYCLIInterval(unsigned C) { StagedAttrs.SYCLIInterval = C; }
 
-  /// Set flag of max_concurrency for the next loop pushed.
-  void setSYCLMaxConcurrencyEnable() {
-    StagedAttrs.SYCLMaxConcurrencyEnable = true;
-  }
-
-  /// Set value of threads for the next loop pushed.
-  void setSYCLMaxConcurrencyNThreads(unsigned C) {
+  /// Add value of max_concurrency for the next loop pushed.
+  void addSYCLMaxConcurrencyNThreads(unsigned C) {
```

Comment:
Sorry, but now these are all `set` again rather than `add` (the only real reason for add was because I thought we were allowing multiple values in a container).

Reply:
> Sorry, but now these are all `set` again rather than `add` (the only real reason for add was because I thought we were allowing multiple values in a container).

The attribute take 0, 1, ... values but as a single value.

Reply:
Done